roles/openshift_master/tasks/main.yml

```diff
@@ -171,6 +171,14 @@
   when: openshift_master_ha | bool and openshift_master_cluster_method == "native"
 # end workaround for missing systemd unit files
 
+# If the session secrets file was missing from any master, we should
+# clean up the file on masters so that we can get them in sync.
+- name: Delete session secrets file
+  file:
+    path: "{{ openshift.master.session_secrets_file }}"
+    state: absent
+  when: session_secrets_missing | bool
+
```

Comment:
Actually, never mind that last comment, since the secrets would already match the values on disk for the first master, so setting the contents would be a noop on the first master anyway.